### PR TITLE
Merge Draco 6-22 back to develop

### PR DESCRIPTION
**Pull request by RyanWollaeger**
See changelog.

* [Pre-Merge Code Review](https://github.com/lanl/Draco/wiki/Style-Guide)
  * [x] Travis CI checks pass
  * [ ] Code coverage does not decrease
  * [ ] [Valgrind test passes](https://rtt.lanl.gov/CDash/index.php?project=Draco)
  * [x] [Toss2 checks pass](https://rtt.lanl.gov/CDash/index.php?project=Draco)
  * [x] [Toss3 checks pass](https://rtt.lanl.gov/CDash/index.php?project=Draco)
  * [ ] [Trinitite checks pass](https://rtt.lanl.gov/CDash/index.php?project=Draco)
  * [ ] Code reviewed/approved, sufficient DbC checks, testing, documentation

### CODE REVIEW

File: regression/scripts/release_toss2.sh [hunk 1]
```diff
@@ -81,15 +80,29 @@ establish_permissions
...
+# ppn=`showstats -n | tail -n 1 | awk '{print $3}'`
 #build_pe=`npes_build`
 #test_pe=`npes_test`
 
-avail_queues=`mdiag -u $LOGNAME | grep ALIST | sed -e 's/.*ALIST=//' | sed -e 's/,/ /g'`
+# SLURM
+avail_queues=`sacctmgr -np list assoc user=$LOGNAME | sed -e 's/.*|\(.*dev.*\|.*access.*\)|.*/\1/' | sed -e 's/|.*//'`
+
 case $avail_queues in
   *access*) access_queue="-A access" ;;
```
Comment (KineticTheory) [inline regression/scripts/release_toss2.sh]: This should be 
```
access_queue="-A access --qos=access"
```

The incorrect version doesn't impact the release - it just means that the tests took longer to start.

File: regression/scripts/release_toss2.sh [hunk 2]
```diff
@@ -163,8 +176,10 @@ for env in $environments; do
 
     # export dry_run=1
     export steps="config build test"
-    cmd="msub -V $access_queue -l walltime=01:00:00 -l nodes=1:ppn=${ppn} -j oe \
--o $source_prefix/logs/release-$buildflavor-$version.log $script_dir/release_toss2.msub"
+    cmd="sbatch -J release_draco $access_queue -t 1:00:00 -N 1 \
+-o $source_prefix/logs/release-$buildflavor-$version.log \
+-e $source_prefix/logs/release-$buildflavor-$version.log \
```
Comment (KineticTheory) [inline regression/scripts/release_toss2.sh]: The `-e $logfile` isn't required.

File: regression/scripts/release_toss2.sh (hunk 1 above)
Comment (RyanWollaeger) [inline regression/scripts/release_toss2.sh]: This has been added.

File: regression/scripts/release_toss2.sh (hunk 2 above)
Comment (RyanWollaeger) [inline regression/scripts/release_toss2.sh]: This has been removed.

File: regression/scripts/release_cray.sh [hunk 3]
```diff
@@ -98,15 +98,15 @@ export TARGET=knl
 ##---------------------------------------------------------------------------##
 sdir=`dirname $0`
 cdir=`pwd`
```
Comment (alexrlongne) [inline regression/scripts/release_cray.sh]: I think everything looks good. One comment that shouldn't hold up the merge: it'd be nice if some of these variables had more descriptive names. What are "sdir" and "cdir"?

**Comment by KineticTheory** [comment]
@alexrlongne I eliminated the variable `sdir` and I renamed `cdir` to `initial_working_dir`.

**Pull request merged**